pkgs/development/python-modules/textnets/default.nix

```diff
@@ -59,10 +63,22 @@ buildPythonPackage rec {
     "textnets"
   ];
 
+  # Enables the package to find the cythonized .so files during testing. See #255262
+  preCheck = ''
+    python setup.py build_ext --inplace
+  '';
+
   disabledTests = [
-    # Test fails: A warning is triggered because of a deprecation notice by pandas.
-    # TODO: Try to re-enable it when pandas is updated to 2.1.1
-    "test_corpus_czech"
+    # Test fails: Throws a UserWarning asking the user to install `textnets[fca]`.
+    "test_context"
+  ];
+
+  pytestFlagsArray = [
+    # Test fails: These warnings break the build.
+    "-W"
+    "ignore::DeprecationWarning"
+    "-W"
+    "ignore::FutureWarning"
```

Comment:
```suggestion
```
Could you please check again if these flags are needed?
I removed them, but no error was thrown.

Reply:
Thank you so much for your patient help. This is only my second nixpkgs PR to get to review.